.github/workflows/po-linter.yaml

```diff
@@ -0,0 +1,54 @@
+name: Prometheus Operator Linter
+
+on:
+  pull_request:
+    paths:
+    - namespaces/live.cloud-platform.service.justice.gov.uk/*/*prometheus.yaml
+    - namespaces/live.cloud-platform.service.justice.gov.uk/*/*prometheus.yml
+    - namespaces/live-2.cloud-platform.service.justice.gov.uk/*/*/*prometheus.yaml
```

Comment:
Should this be `namespaces/live-2.cloud-platform.service.justice.gov.uk/*/*prometheus.yaml